docs/labels.md

```diff
@@ -0,0 +1,106 @@
+# GitHub Label List
```

Comment:
Name this file issue-labels.md or github-labels.md?

Reply:
I think `github-labels.md` is appropriate. Especially since labels is ambiguous in the Kubernetes domain. This allows it to address labeling both GitHub issues and PRs.